### ISSUE TITLE
Multiple Issues with 2.7.0.0

### ISSUE THREAD

**Issue by unL33T123** [phw198/OutlookGoogleCalendarSync#668]
:memo: Use the below template to report your issue.  
:bulb: Upgrade to the latest alpha release to check if the problem is already fixed.  
:paperclip: To aid investigation, provide your OGcalsync.log file ([see instructions](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file)).  

----

**OGCS Version**: v2.7.0.0
**Installed or Portable**: Portable

**Problem Description**  
1. Changed Gmail accounts.  Despite sync setting being on Merge all items from Outlook were deleted.
2. Authentication does not work.  After allowing OGCS access I get a "This page cannot be displayed message".
3. Will not sync due to above.  Constant "Failed to retrieve Google event."
4. Stop Sync button does nothing.
5. Clicking Disconnect Account when account is not connected causes crash.

**Steps to Reproduce the Issue**  
1. Change Gmail account.  All items will be removed from Outlook.
2. a. Click Disconnect Account.  Then click Retrieve Calendars.
b. Enter login details.
c. Allow OGCS access.
d. Get presented with "Page cannot be displayed." when browser attempts to connect to localhost.
3. Attempt to sync after closing browser window from 2.d.  Will get error message.
4. Press Stop Sync while it is syncing.  It will continue to pop up errors and not stop syncing.
5. Click Disconnect Account when one is not set up or click it more than once and program will crash.

**Any other information**  
\Everything was working fine before I tried to change accounts.
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/2894273/OGcalsync.log)

**Comment by phw198** [comment]
So the "merge" option before v2.7.7 will only merge OGCS synced items into a calendar with pre-existing non-OGCS synced items. If you sync items to a calendar, then change the source calendar (or Google account) and sync to the same target calendar, yes, it would wipe all the previously synced items. 

From v2.7.7 OGCS supports syncing multiple calendars into the same target calendar.

Your authorisation issues sound related to you having accessed websites on `https://localhost` before? See [these tweets](https://twitter.com/OGcalsync/status/1088454391688908801) for someone else who had the problem. Perform a hard refresh on the localhost "page cannot be displayed" screen, use incognito mode, or a different browser entirely to complete the authorisation process successfully.

I can't replicate problem 5 on alpha v2.7.8 - please [upgrade](https://github.com/phw198/OutlookGoogleCalendarSync/releases/download/v2.7.8-alpha/Portable_OGCS_v2.7.8.zip) and let me know if problems continue.

**Comment by phw198** [comment]
I've now [added a section to the Wiki FAQ page](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/FAQs#page-cannot-be-displayed-after-authorising-ogcs-to-access-my-calendar) - let me know how you fare.

**Comment by unL33T123** [comment]
I ended up trying 2.7.7.0 when I ran into my problems and it has been working much better.

Only problem I've noticed is if I close Outlook it gives my an error syncing whereas the other version didn't.  That could also be caused by a change in Outlook, though.  Not a big deal at all.

Thank you for your response.  I guess I was under the impression the non-alpha versions should be working better.  2.7.7.0 is working fine for me for now but if I run into additional issues I will upgrade before posting.
